documentation/modules/deploying/proc-deploy-migrate-kraft.adoc

```diff
@@ -0,0 +1,257 @@
+// Module included in the following assemblies:
+//
+// deploying/deploying.adoc
+
+[id='proc-deploy-migrate-kraft-{context}']
+= Migrating to KRaft mode
+
+[role="_abstract"]
+If you are using ZooKeeper for metadata management in your Kafka cluster, you can migrate to using Kafka in KRaft mode. 
+KRaft mode replaces ZooKeeper for distributed coordination, offering enhanced reliability, scalability, and throughput.
+
+During the migration, you install a quorum of controller nodes as a node pool, which replaces ZooKeeper for management of your cluster. 
+You enable KRaft migration in the cluster configuration by applying the `strimzi.io/kraft: migration` annotation.  
+After the migration is complete, you switch the brokers to using KRaft and the controllers out of migration mode using the `strimzi.io/kraft: enabled` annotation.
+
+Before starting the migration, verify that your environment can support Kafka in KRaft mode, as there are a number of xref:ref-operator-use-kraft-feature-gate-str[limitations].
+Note also, the following:
+
+* Migration is only supported on dedicated controller nodes, not on nodes with dual roles as brokers and controllers.
+* Throughout the migration process, ZooKeeper and Controller nodes operate in parallel for a period, requiring sufficient compute resources in the cluster.
+
+WARNING: ZooKeeper to KRaft migration is not yet suitable for production environments.
+
+.Prerequisites
+
+* You must be using Strimzi 0.40 or newer with Kafka 3.6.1 or newer. If you using an earlier version of Strimzi or APache Kafka, upgrade before migrating to KRaft mode.
+* The Cluster Operator that manages the Kafka cluster is running.
+* The Kafka cluster deployment uses Kafka node pools.
++
+You can xref:proc-migrating-clusters-node-pools-str[migrate your existing Kafka cluster to use node pools]. 
++
+In the `KafkaNodePool` resource configuration, brokers must be contained in a node pool assigned a `broker` role and with the name `kafka`.
```

Comment:
"and with the name `kafka`" ... from where did you get this "restriction"? We don't have that, the node pool can have the name you prefer (I have been using "broker" as name for the node pool for example, since long time).

Reply:
This is what we have as a requirement when creating the broker node pool: https://strimzi.io/docs/operators/in-development/deploying#proc-migrating-clusters-node-pools-str

Reply:
But it's not valid when a user has a Kafka cluster which is using ZooKeeper and was created with node pools seems the beginning.
So I think we have two situation here:
1. a user created a Kafka cluster with ZooKeeper but using node pools since beginning and they want to migrate to KRaft now --> their brokers node pool is named as they want
2. a user created a Kafka cluster with ZooKeeper without node pools. They want to migrate but they need to move to use node pools first --> what you are saying applies (following the link you provided first and then KRaft migration).

Reply:
I've put the paragraph together and adjusted as follows:

```
You can xref:proc-migrating-clusters-node-pools-str[migrate your existing Kafka cluster to use node pools]. 
To migrate, the `KafkaNodePool` resource configuration, brokers must be contained in a node pool assigned a `broker` role and with the name `kafka`.
Support for node pools is enabled in the `Kafka` resource configuration using the `strimzi.io/node-pools: enabled` annotation.
```

Reply:
I think the prerequisite here is just you have to have a Kafka cluster ZooKeeper-based using node pools.
If it's already using nodepool for brokers, you are ok, you can start to migrate to KRaft.
If not, you have to migrate to use node pools first (as the link we have) then you can start the KRaft migration.
Can we say something like this? It looks clearer to me.

Reply:
Sure. I've updated to be clearer about this